### ISSUE TITLE
(minor) DUODataUse "modifiers" property definition doubt.

### ISSUE THREAD

**Issue by redmitry** [ga4gh-beacon/beacon-v2-Models#99]
Hello,

There is a strange construct in the [datasets/defaultSchema.json](https://github.com/ga4gh-beacon/beacon-v2-Models/blob/e11b14a328bba7888ee218fb750e53c14f7edd6d/BEACON-V2-Model/datasets/defaultSchema.json#L94)
~~~json
"modifiers": {
    "type": "array",
    "items": {
        "allOf": [
            {
                "$ref": "https://raw.githubusercontent.com/ga4gh-beacon/beacon-framework-v2/main/common/ontologyTerm.json"
            },
            {
                "examples": [
                    {
                        "id": "EFO:0001645"
                    },
                    {
                        "id": "EFO:0001655"
                    }
                ]
            }
        ]
    }
}
~~~
IMHO could be more clear:
~~~json
"modifiers": {
    "type": "array",
    "items": {
        "$ref": "https://raw.githubusercontent.com/ga4gh-beacon/beacon-framework-v2/main/common/ontologyTerm.json"
     },        
     "examples": [
         [ { "id": "EFO:0001645" }, { "id": "EFO:0001655" } ],
         [ { "id": "EFO:0001665" }, { "id": "EFO:0001675" } ]
      ]
}
~~~

Note that while there is no validation rules for the example keyword, for instance in the [datasets/defaultSchema.json](https://github.com/ga4gh-beacon/beacon-v2-Models/blob/e11b14a328bba7888ee218fb750e53c14f7edd6d/BEACON-V2-Model/biosamples/defaultSchema.json#L100) IMO it should be an array also.

Cheers,

Dmitry

**Comment by mbaudis** [comment]
@redmitry Good catch. This is a remnant from earlier JSON Schema versions which didn’t allow any other argument (examples, definitions…)when using a `$ref`. 

AFIK

**Comment by jrambla** [comment]
Solved in #100
